africanlii/settings.py

```diff
@@ -3,3 +3,7 @@
 INSTALLED_APPS = ["africanlii.apps.AfricanliiConfig"] + INSTALLED_APPS  # noqa
 
 ROOT_URLCONF = "africanlii.urls"
+
+JAZZMIN_SETTINGS["site_title"] = "Africanlii"  # noqa
```

Comment:
All three of these can default to `PEACHJAM["APP_NAME"]` and be set in peachjam.settings so that they don't have to be set here. More automatic, less manual :)